packages/fxa-settings/src/pages/Index/container.tsx

```diff
@@ -113,7 +130,7 @@ export const IndexContainer = ({
       try {
         if (!isEmailValid(email)) {
```

Comment:
Use the previous validation here if email comes from query param?

Reply:
I'm guessing by 'use' you are thinking that we should try to invoke the same validation that the @IsEmail() validator would?

If so I agree with that, and one thing to note is each class-validator has a corresponding function that can be directly invoked. e.g.

```

import {IsEmail, isEmail} from 'class-validator';

class O {
   @IsEmail()
   email:string
}


// Email could be directly validated by calling
isEmail(o.email)
```

Reply:
I replaced to directly use `isEmail` from the `class-validator`